components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -273,4 +273,21 @@ - (void)testAccessibilityIdentifiersAppliesToAlertControllerViewButtons {
   XCTAssertEqualObjects(button2.accessibilityIdentifier, @"A");
 }
 
+- (void)testCornerRadius {
+  // Given
+  CGFloat cornerRadius = 36.0;
```

Comment:
Please explicitly cast `36.0` to (CGFloat) or we are likely to get errors when compiling for 32-bit architectures. Implicit loss of precision is configured as an error for MDC-iOS.

Alternatively, you can just use the integral literal `36` since that can implicitly convert to single- and double-precision floats without loss of precision.